test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.cs

```diff
@@ -4,130 +4,26 @@
 using System;
 using System.Buffers;
 using System.Collections.Generic;
-using System.Net.Http;
-using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Client.Tests;
 using Microsoft.AspNetCore.SignalR.Internal;
 using Microsoft.AspNetCore.SignalR.Internal.Protocol;
 using Microsoft.AspNetCore.SignalR.Tests.Common;
-using Microsoft.AspNetCore.Sockets;
 using Microsoft.AspNetCore.Sockets.Client;
-using Microsoft.AspNetCore.Sockets.Client.Tests;
 using Microsoft.Extensions.Logging;
 using Moq;
-using Moq.Protected;
 using Xunit;
 
 namespace Microsoft.AspNetCore.SignalR.Client.Tests
 {
     public class HubConnectionTests
     {
         [Fact]
-        public void CannotCreateHubConnectionWithNullUrl()
-        {
-            var exception = Assert.Throws<ArgumentNullException>(
-                () => new HubConnection((Uri)null, Mock.Of<ILoggerFactory>()));
-            Assert.Equal("url", exception.ParamName);
-        }
-
-        [Fact]
-        public async Task CanDisposeNotStartedHubConnection()
```

Comment:
I'll add tests that verify those call the underlying connection start/disposeasync